hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -249,6 +262,128 @@ public String compact(
     return "Compaction successfully completed for " + compactionInstantTime;
   }
 
+  /**
+   * Prints all compaction details.
+   * Note values in compactionsPlans and instants lists are expected to be in cohesion
```

Comment:
what does this comment mean ?

Reply:
its no longer relevant. so removed it. I intially had two parameters: List<Instant> and List<CompactionPlan>. But that didnt seem good, so changed it to one parameter.